Makefile

```diff
@@ -14,20 +14,16 @@ build: packr-build compile-local packr-clean
 release: packr-build compile-darwin compile-linux compile-windows packr-clean
 
 compile-local:
-	@echo "Building archer to ./bin/local/archer" &&\
-	CGO_ENABLED=0 go build -o ./bin/local/archer ./cmd/archer
+	PLATFORM=local DESTINATION=./bin/local/archer ./scripts/build_binary.sh
```

Comment:
Do we need a `CGO_ENABLED=0` here as well?

Reply:
NO